tools/bin/hawqpylib/hawqlib.py

```diff
@@ -203,6 +203,13 @@ def check_hostname_equal(remote_host, user = ""):
     cmd = "hostname"
     result_local, local_hostname, stderr_remote  = local_ssh_output(cmd)
     result_remote, remote_hostname, stderr_remote = remote_ssh_output(cmd, remote_host, user)
+    if result_remote != 0:
+        print "Execute command '%s' failed with return code %d on %s." % (cmd, result_remote, remote_host)
+        print "Either ssh connection fails or command exits with error. Details:"
+        print stderr_remote
+        print "For ssh connection issue, please make sure passwordless ssh is enabled or check remote host."
```

Comment:
Although a simple command, it's possible the remote OS have some issue can cause failures. So not only the ssh connection error.